### ISSUE TITLE
libpng warning incorrect sRGB profile

### ISSUE THREAD

**Issue by patrickdupuis** [supercollider/supercollider#3104]
I get these warnings when I open scide from the command line:
```
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
```
A google search led me to this post: https://stackoverflow.com/questions/22745076/libpng-warning-iccp-known-incorrect-srgb-profile#22747902

> Libpng-1.6 is more stringent about checking ICC profiles than previous versions. You can ignore the warning. To get rid of it, remove the iCCP chunk from the PNG image.

The solution seems to be running ImageMagick's `mogrify` command on the two offending .png files. I'm not certain which files are causing the warning. It sounds like we could run `mogrify` on all png files in the repository and it would do the trick.

**Comment by brianlheim** [comment]
Thanks for the research @patrickdupuis !

**Comment by patrickdupuis** [comment]
if you can help me figure out how to run that command on all relevant files, i can try to take care of this.

**Comment by brianlheim** [comment]
It looks like you can use [pngcrush](https://en.wikipedia.org/wiki/Pngcrush) with the option `-rem iCCP`.

**Comment by patrickdupuis** [comment]
I gave this a shot.
 
Tried `find . -name '*.png' -exec mogrify {} \;` and `find . -name '*.png' -exec pngcrush -ow -rem iCCP {} \;`.

After re-building SC, no change. Maybe the above worked, but I was testing it wrong. I don't know...

**Comment by patrickdupuis** [comment]
Tried this again with no luck. `pngcrush` gives some interesting output on each of the files. I can see that it is doing something. I have been avoiding the png files that are located in `/external_libraries/*`. Maybe that's were the offending files are located?

**Comment by brianlheim** [comment]
according to `pngcrush -n -q` (suggested in the SO post), the bad file is `HelpSource/Other/ccbysa3_88x31.png`.

**Comment by brianlheim** [comment]
@patrickdupuis - can you check out the issue/3104 branch and see if you still get this?

**Comment by patrickdupuis** [comment]
Yes, I still get the warnings on this branch. There should be 2 files that need changing since the message appears twice at startup. I wish there was a way to see where these warnings are coming from. I tried a debugger and it didn't give me any additional information.

**Comment by brianlheim** [comment]
> There should be 2 files that need changing since the message appears twice at startup.

Could be one file that triggers the warning twice. If you remove that file from the build altogether, do you still get that warning?

**Comment by patrickdupuis** [comment]
Unfortunately, I still get the warning if I remove that file.

**Comment by patrickdupuis** [comment]
This issue may no longer be present on current `develop`, possibly due to the Qt update. I will confirm this on another distro just to be sure.

**Comment by shimpe** [comment]
FWIW: No longer occurs on linux (arch linux - qt 5.11.1)
